tests/cc_haskell_import/BUILD

```diff
@@ -55,7 +55,10 @@ cc_binary(
     linkstatic = 1,
     tags = ["requires_threaded_rts"],
     visibility = ["//tests:__subpackages__"],
-    deps = ["@ghc//:threaded-rts"],
+    deps = [
+        "@ghc//:threaded-rts",
+        "@gmp",
```

Comment:
Does `cc_haskell_import` not forward `cc_library` dependencies?

Reply:
I'm not sure what you mean

Reply:
Nvm, thought `hs-lib-b.so` was in `deps`, but it's in `srcs`.